### PR TITLE
workflows: add lint

### PR DESCRIPTION
**Pull request by EncryptedDev**
Co-Authored-By: Rodry <38259440+ImRodry@users.noreply.github.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "license": "MPL-2.0",
   "scripts": {
     "lint": "eslint --fix . && prettier --write **/*.ts **/{metadata,tsconfig}.json",
+    "lint:ci": "eslint . && prettier **/*.ts **/{metadata,tsconfig}.json",
```
Comment (Slowlife01) [inline package.json:8]: Is prettier necessary 🤔

File: .github/workflows/SE.yml [hunk 2]
```diff
@@ -14,8 +14,7 @@ jobs:
         uses: actions/setup-node@v2.5.0
         with:
           node-version: 16
-          cache: "yarn"
```
Comment (ImRodry) [inline .github/workflows/SE.yml:17]: this can be undone

File: .github/workflows/SV.yml [hunk 3]
```diff
@@ -18,8 +18,7 @@ jobs:
         uses: actions/setup-node@v2.5.0
         with:
           node-version: 16
-          cache: "yarn"
```
Comment (ImRodry) [inline .github/workflows/SV.yml:21]: this can be undone

File: .github/workflows/SE.yml (hunk 2 above)
Comment (EncryptedDev) [reply .github/workflows/SE.yml:17]: Boi

File: package.json [hunk 4]
```diff
@@ -5,6 +5,7 @@
   "license": "MPL-2.0",
   "scripts": {
     "lint": "eslint --fix . && prettier --write **/*.ts **/{metadata,tsconfig}.json",
+    "lint:ci": "eslint .",
```
Comment (ImRodry) [inline package.json:8]: rhys do you ever listen to me
```suggestion
    "lint:ci": "eslint . && prettier **/*.ts **/{metadata,tsconfig}.json",
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: .github/workflows/SE.yml (hunk 2 above)
Comment (ImRodry) [reply .github/workflows/SE.yml:17]: what boi

File: package.json (hunk 1 above)
Comment (ImRodry) [reply package.json]: what

File: .github/workflows/SE.yml (hunk 2 above)
Comment (Bas950) [reply .github/workflows/SE.yml:17]: can be undone

File: .github/workflows/SV.yml (hunk 3 above)
Comment (Bas950) [reply .github/workflows/SV.yml:14]: can be undone

File: .github/workflows/lint.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v2
+      - name: Use Node.js 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
```
Comment (ImRodry) [inline .github/workflows/lint.yml:13]: https://github.com/actions/cache/issues/698
```suggestion
      - name: Use Node.js 16
        uses: actions/setup-node@v2
        with:
          node-version: 16
          cache: "yarn"
```

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**